include/micm/solver/cuda_linear_solver.cuh

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) 2023 National Center for Atmospheric Research,
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+#include <micm/util/cuda_param.hpp>
+#include <vector>
+namespace micm
+{
+  namespace cuda
+  {
+    std::chrono::nanoseconds SolveKernelDriver(
+     CudaLinearSolverParam& linearSolver,
+     CudaSparseMatrixParam& sparseMatrix, 
+     CudaMatrixParam& denseMatrix);
```

Comment:
Could you explain why here needs the input of both sparse and dense matrix?

Reply:
The parameters of Solve function x, b, lowerMatrix, upperMatrix. x and b are dense matrix; lowerMatrix and upperMatrix are sparse matrix.

Reply:
Interesting. why is upperMatrix a sparse matrix but upperMatrix.x a dense matrix? And what is the difference between upperMatrix and upperMatrix.x?

Reply:
Sorry. There is no such variable upperMatrix.x. It was a period. 
So, 
x, b are dense matrix. 
upperMatrix, lowerMatrix is sparse matrix.

Reply:
Aha, I see. I misunderstood your statement. That is clear to me now. Thanks.